model/src/main/kotlin/utils/FileArchiver.kt

```diff
@@ -144,6 +144,14 @@ private fun getArchivePath(provenance: Provenance): String =
 private val SHA1_DIGEST by lazy { MessageDigest.getInstance("SHA-1") }
 
 /**
- * Calculate the SHA-1 hash of the string representation of this [Provenance] instance.
+ * Calculate the SHA-1 hash of the storage key of this [Provenance] instance.
```

Comment:
> Previosuly the used key inappropriately incorporated additonal attributes like e.g., the download time

IIRC, taking the download time into account was a deliberate design decision by @mnonnenmacher, and the argument was that e.g. source artifacts on Maven Central might sneakily be changed over time while maintaining their coordiantes. But thinking about this again, the `RemoteArtifact` also contains the hash, so we should be fine, right?

Reply:
@mnonnenmacher  are you fine with this change in above mentioned regard?

Reply:
@sschuberth have you noticed that using the download time in the hash would turn the time to look-up from O(1) into O(n) ?

Reply:
Thinking about it again, taking the *download* time into account here indeed does not make any sense at all to me. The last *modified* time would have made some sense, but that should also not be necessary as we have the file hash.

Reply:
My point with O(n) does not apply only to 'download' but to any attribute which is not included in the attributes used for lookup. So it's not just not necessary, but a really bad idea to add further attributes.

Reply:
Yes, I'm fine with that. Actually, I don't remember if it was intentional to include the download time.